patches/api/0072-Add-PlayerArmorChangeEvent.patch

```diff
@@ -28,9 +28,9 @@ index 0000000000000000000000000000000000000000..e406ce639a2e88b78f82f25e71678a66
 +import static org.bukkit.Material.*;
 +
 +/**
-+ * Called when the player themselves change their armor items
++ * Called when the player's armor is changed(manual armor change, equip through dispenser, loss of durability, etc.)
 + * <p>
-+ * Not currently called for environmental factors though it <strong>MAY BE IN THE FUTURE</strong>
++ * This event does <strong>not</strong> fire when a player joins the server with armor equipped.
```

Comment:
Is this clarification needed? I wouldn't expect this to fire there anyways, so I'm not sure what this is clarifying.

Reply:
it's likely there to document the change in functionality, though as this behavior was undocumented i _think_ it's fine just to leave out

Reply:
I was on the fence about putting it in, on the _very small_ edge case that someone use it on player join, but that is a good point.